apps/dotcms-ui/src/app/portlets/shared/dot-content-types-listing/dot-content-types.component.html

```diff
@@ -1,23 +1,31 @@
 <dot-listing-data-table
-    *ngIf="showTable"
     #listing
     (rowWasClicked)="editContentType($event)"
+    *ngIf="showTable"
     [actionHeaderOptions]="actionHeaderOptions"
-    [paginatorExtraParams]="paginatorExtraParams"
+    [actions]="rowActions"
     [columns]="contentTypeColumns"
+    [paginatorExtraParams]="paginatorExtraParams"
     sortField="modDate"
     sortOrder="DESC"
     url="v1/contenttype"
-    [actions]="rowActions"
 >
     <dot-base-type-selector
-        *ngIf="!filterBy"
         (selected)="changeBaseTypeSelector($event)"
+        *ngIf="!filterBy"
     ></dot-base-type-selector>
 </dot-listing-data-table>
 
 <dot-add-to-bundle
+    (cancel)="addToBundleIdentifier = null"
     *ngIf="addToBundleIdentifier"
     [assetIdentifier]="addToBundleIdentifier"
-    (cancel)="addToBundleIdentifier = null"
 ></dot-add-to-bundle>
+
+
+<dot-dot-content-type-clone-dialog
+    (cancel)="hideCloneContentTypeDialog()"
+    *ngIf="isVisibleCloneDialog$ | async"
+    [assetSelected]="assetSelected$ | async"
+    [isVisibleDialog]="isVisibleCloneDialog$ | async">
+</dot-dot-content-type-clone-dialog>
```

Comment:
Isn't the idea of using `DynamicDialogModule` precisely not to have a component in the DOM at all?

Reply:
Changed to load dynamically.